src/ralph/back_office/templates/backofficeasset/bo_asset_licence.html

```diff
@@ -0,0 +1,39 @@
+{% extends BASE_TEMPLATE %}
```

Comment:
Can't we use Django admin somehow? For example show list based on model and fields chosen in class definition instead of template ? Basically this list should (?) be similar to list od all licences (filtered by asset)